examples/rust_caster.rs

```diff
@@ -264,16 +264,43 @@ fn play_media(
     }
 }
 
+
+fn discover() -> Option<(String, u16)> {
+    use mdns_sd::{ServiceDaemon, ServiceEvent};
```

Comment:
__issue:__ looks like you forgot to include `mdns-sd = "0.6.0"` to `[dev-dependencies]` in `Cargo.toml`. Can you please include changes in `Cargo.toml` and `Cargo.lock` in the PR as well?